jormungandr-lib/src/interfaces/settings.rs

```diff
@@ -0,0 +1,31 @@
+use crate::time::SystemTime;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SettingsDto {
+    pub block0_hash: String,
+    pub block0_time: SystemTime,
+    pub curr_slot_start_time: Option<SystemTime>,
+    pub consensus_version: String,
+    pub fees: SettingsFeesDto,
+    pub max_txs_per_block: u32,
+}
+
+impl PartialEq<SettingsDto> for SettingsDto {
+    fn eq(&self, other: &SettingsDto) -> bool {
+        self.block0_hash == other.block0_hash
+            && self.block0_time == other.block0_time
+            && self.consensus_version == other.consensus_version
+            && self.fees == other.fees
+            && self.max_txs_per_block == other.max_txs_per_block
+    }
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct SettingsFeesDto {
+    pub constant: u64,
+    pub coefficient: u64,
+    pub certificate: u64,
```

Comment:
There is already a `LinearFee` object to do the serialize/deserialize for serde in `jormungandr-lib`

It is used here: https://github.com/input-output-hk/jormungandr/blob/3c221c3c6865b19e5964d69e82e8c26bd8181c8b/jormungandr-lib/src/interfaces/block0_configuration/initial_config.rs#L59-L60

and defined there: https://github.com/input-output-hk/jormungandr/blob/3c221c3c6865b19e5964d69e82e8c26bd8181c8b/jormungandr-lib/src/interfaces/block0_configuration/initial_config.rs#L293-L299

It can be moved to its own module within the `interface`